docs/programming-fundamentals/Arrays.md

```diff
@@ -0,0 +1,137 @@
+# Arrays in C
```

Comment:
This is not a good format based on our documentation style. You need to include basic information at the top of the page, such as the document ID (unique for URL), title, sidebar label, and position. You should also include tags for SEO and best practices.